### ISSUE TITLE
Kernel: Ext2FS error when inserting into persistent SQLite database

### ISSUE THREAD

**Issue by Lubrsi** [SerenityOS/serenity#5991]
As reported in #5972.
Reproduction:
```
$ sqlite3
sqlite> .open serenity.db
sqlite> CREATE TABLE hello (friends TEXT);
sqlite> INSERT INTO hello (friends) VALUES (':^)');
Error: disk I/O error
```

This gets printed in the console:
```
[sqlite3(34:34)]: Ext2FSInode[1:3901]::read_bytes(): Failed to read block 2452 (index 2)
```

**Comment by GMTA** [comment]
@Lubrsi This seems to have been fixed by 5001b71c4 as well, can you confirm? (Thanks @awesomekling !)

**Comment by Lubrsi** [comment]
Can confirm, closing.
